src/components/checkbox/_checkbox.scss

```diff
@@ -1,6 +1,7 @@
 @import '../../globals/scss/vars';
 @import '../../globals/scss/theme';
 @import '../../globals/scss/colors';
+@import '../../globals/scss/colors--experimental';
```

Comment:
Do I need to add this in the `button.scss` or will this be available to all components once this PR is merged?

Reply:
this is going to get added to all components